spec/models/miq_policy_spec.rb

```diff
@@ -100,6 +100,16 @@
       end
     end
 
+    describe '#miq_event_definitions' do
+      before do
+        policy.miq_policy_contents.push(FactoryGirl.create(:miq_policy_content))
```

Comment:
``` 
policy.miq_policy_contents.push(miq_policy_content)
```

this needs to be in before .. end as you have it,  it is calling of method not creating ... I am sorry for confusing